src/fastpam1.cpp

```diff
@@ -0,0 +1,193 @@
+/**
+ * @file fastpam1.cpp
```

Comment:
As per [the issue](https://github.com/ThrunGroup/BanditPAM/issues/106), include citations to Schubert's original PAM paper and the FastPAM paper